src/hardware/serialport/softmodem.h

```diff
@@ -230,6 +231,8 @@ class CSerialModem : public CSerial {
 	Bitu plusinc;
 	Bitu cmdpos;
 	Bitu flowcontrol;
+	Bitu dtrmode;
+	Bits dtrofftimer;
```

Comment:
Just a question:  are these types machine memory-bit-depth critical? ie: on 64-bit memory machines, these members needs to 64-bit INTs; and on 32-bit memory-machines, these members need to be 32-bit INTs?

If they're not sensitive to memory sizes, we have been trying to move away from DOSBox's customized wrapper-types in favour of standard types here https://en.cppreference.com/w/cpp/types/integer.

If bog-standard `int` or `uint32_t` will do, then let's use those.  This simplifies maintenance because memory-specific types (like `Bitu`) additionally tell the developers, _"watch out here: this variables type-size is critical for host-specific reasons either for performance or API/emulation compliance"_

Reply:
No, they're not. But I'd rather have the entire modem code updated later rather than edit just these two vars and have them stick out like a sore thumb.

Reply:
Sounds good! 

> I'd rather have the entire modem code updated later

I like where you're going with this.